desc/magnetic_fields/_dommaschk.py

```diff
@@ -478,12 +504,11 @@ def V_m_l(R, phi, Z, m, l, a, b, c, d):
         (same size as the size of the given R,phi, or Z arrays).
 
     """
-    return (a * jnp.cos(m * phi) + b * jnp.sin(m * phi)) * D_m_n(R, Z, m, l) + (
-        c * jnp.cos(m * phi) + d * jnp.sin(m * phi)
+    return (a * sp.cos(m * phi) + b * sp.sin(m * phi)) * D_m_n(R, Z, m, l) + (
+        c * sp.cos(m * phi) + d * sp.sin(m * phi)
     ) * N_m_n(R, Z, m, l - 1)
 
 
-@jit
 def dommaschk_potential(R, phi, Z, ms, ls, a_arr, b_arr, c_arr, d_arr, B0=1):
```

Comment:
the arguments to this are now expected to be sympy variables, not numpy or jax arrays right? Should update the docstring